jenkins/scality-glance-store-functional-tests/extras.d/55-scality-glance-store.sh

```diff
@@ -0,0 +1,28 @@
+# 55-scality-glance-store - Devstack extra script to configure Glance API with Scality store
+
+function install_scality_store {
+    sudo pip install https://github.com/scality/scality-sproxyd-client/archive/master.tar.gz
```

Comment:
Could this be parametrized, so we can have the CI job work on several branches/releases/...?